extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayRuntimeConfig.java

```diff
@@ -21,6 +21,13 @@ public FlywayDataSourceRuntimeConfig getConfigForDataSourceName(String dataSourc
         return namedDataSources.getOrDefault(dataSourceName, FlywayDataSourceRuntimeConfig.defaultConfig());
     }
 
+    /**
+     * Flag to enable / disable Flyway.
+     *
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
```

Comment:
When will this be used?

Reply:
It's used to turn flyway off in the main container, since the flyway init is externalized.